### PR TITLE
Cleanup docker tutorial

### PR DESCRIPTION
**Pull request by kprosise**
The Docker tutorial section had a number of style and grammar fixes applied. Information was also updated to reflect the usage of the Fast First Target, so mentions of build wait time was removed, as well as the separate branches.

QA Steps: Content was reviewed for correctness where possible, however a detailed follow along was skipped. Linter was ran. HTML output was checked for correct rendering. No issues were spotted.

This commit addresses FFTK-2511
This commit applies to FFTK-2510
This commit applies to FFTK-2430
This commit applies to FFTK-988

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

see commit comment and related issues.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 09c2b147cb565e367066b26cbce4c7ff5b4d833b are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2218/docs/artifacts/html/index.html

File: source/tutorials/getting-started-with-docker/gs-httpd.rst [hunk 1]
```diff
@@ -30,6 +30,4 @@ Check the content of your ``httpd.sh``:
       echo "= $(date) ============================="
      done
 
-This is a shell script file that will respond to a request on the port defined by the 
-PORT environment variable (defaults to ``8080``) with the message defined by the MSG 
-environment variable (defaults to ``OK``).
\ No newline at end of file
+This script responds to a request on the port defined by the ``PORT`` environment variable (defaulting to ``8080``) with the message defined by the ``MSG`` environmental variable (defaults to ``OK``).
```
Comment (kprosise) [inline source/tutorials/getting-started-with-docker/gs-httpd.rst:33]:
```suggestion
This script responds to a request on the port defined by ``PORT``(defaults to ``8080``) with the message defined by ``MSG``(defaults to ``OK``).
```

**Review by kprosise — COMMENTED** [review]
Shorten long sentence.

**Comment by doanac** [comment]
Docs for 6f157b33de4174b682fe7715b69d4a8444e3df5b are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2221/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]
LGTM

File: source/tutorials/getting-started-with-docker/gs-docker-compose.rst [hunk 2]
```diff
@@ -1,13 +1,11 @@
-docker-compose.yml
-^^^^^^^^^^^^^^^^^^
+Docker Compose YAML
+^^^^^^^^^^^^^^^^^^^
 
-This is a YAML file defining services, networks, and volumes for multi-container 
-Docker applications. In other words, all the parameters you have used with 
-``docker run`` you could specify in a ``docker-compose.yml`` file. Then, with a 
-single command, create and start all the services with your configurations.
+The YAML file ``docker-compose.yml`` defines services, networks, and volumes for multi-container Docker applications.
```
Comment (angolini) [inline source/tutorials/getting-started-with-docker/gs-docker-compose.rst:4]: isn't this like too long?

**Review by angolini — APPROVED** [review]
LGTM, I only think that some lines could be shorter, but no big deal

File: source/tutorials/getting-started-with-docker/gs-docker-compose.rst (hunk 2 above)
Comment (kprosise) [reply source/tutorials/getting-started-with-docker/gs-docker-compose.rst:4]: Yes and no. There are a couple rules/suggestions that influence line length. Semantic newlines with sentences less than 25 words > column 100. So if semantic newlines are not used, then column 100 is the cutoff.

Comment (kprosise) [reply source/tutorials/getting-started-with-docker/gs-docker-compose.rst:4]: But I do see a way to shorten the sentence, which I will do.

Comment (kprosise) [inline source/tutorials/getting-started-with-docker/gs-docker-compose.rst:4]:
```suggestion
``docker-compose.yml`` defines the services, networks, and volumes for multi-container Docker apps.
```

File: source/tutorials/getting-started-with-docker/gs-docker-compose.rst [hunk 3]
```diff
@@ -41,15 +39,11 @@ Read the ``docker-compose.yml`` file:
            MSG: "${MSG-Hello world}"
 
 Most of the parameters were already used in the previous commands. 
-The only thing you need to change is the image parameter.
+The only thing to change is the image parameter.
 
-In the next tutorial, you will build and deploy the image with 
-FoundriesFactory and there the image with ``hub.foundries.io`` will be necessary.
+In the next tutorial, you will build and deploy the image with FoundriesFactory®, where the image with ``hub.foundries.io`` will be necessary.
```
Comment (kprosise) [inline source/tutorials/getting-started-with-docker/gs-docker-compose.rst:44]:
```suggestion
In the next tutorial, you will build and deploy with FoundriesFactory®, where ``hub.foundries.io`` will be necessary.
```

File: source/tutorials/getting-started-with-docker/gs-docker-compose.rst [hunk 4]
```diff
@@ -41,15 +39,11 @@ Read the ``docker-compose.yml`` file:
...
 
-For now, because you are still developing locally, you need to 
-edit the image parameter to use the image you have built in the previous steps.
-
-Change the image parameter to the name and tag we built locally ``shellhttpd:1.0``:
+As you are still developing locally, you need to edit the image parameter to use the image and tag built during the previous steps, ``shellhttpd:1.0``:
```
Comment (kprosise) [inline source/tutorials/getting-started-with-docker/gs-docker-compose.rst:46]:
```suggestion
As you are still developing locally, edit the image parameter to use the image and tag from the previous steps, ``shellhttpd:1.0``:
```

File: source/tutorials/getting-started-with-docker/gs-summary.rst [hunk 5]
```diff
@@ -1,12 +1,10 @@
 Summary
 -------
-You have learned the basic commands and structure of Docker and ``docker-compose``
-applications.  At this point, if you just created your Factory, the initial build 
-should be finished. In the next tutorial, you will commit and push your changes 
-to the remote ``containers.git`` repository.  FoundriesFactory CI will then build 
-and deploy this application as a **Target** which can be applied to your device.
+You have learned the basic commands and structure of Docker and ``docker-compose`` applications.
+In the next tutorial, you will commit and push your changes to the remote ``containers.git`` repository.
+The FoundriesFactory® CI will then build and deploy this application as a **Target** which can be applied to your device.
```
Comment (kprosise) [inline source/tutorials/getting-started-with-docker/gs-summary.rst:5]:
```suggestion
The FoundriesFactory® CI will build and deploy this app as a **Target** which can be applied to your device.
```

File: source/tutorials/getting-started-with-docker/getting-started-with-docker.rst [hunk 6]
```diff
@@ -1,12 +1,11 @@
...
+Getting Started With Docker
 ===========================
 
-FoundriesFactory gives you the ability to over-the-air update ``docker-compose`` 
-applications.  This tutorial contains step-by-step instructions on how to get 
-started with Docker and ``docker-compose`` applications, including the basic 
-commands and concepts to help you to create your own application.
+FoundriesFactory® gives you the ability to update ``docker-compose`` applications Over The Air (OTA).
+This tutorial gets you started with Docker and ``docker-compose``,
+with the basic commands and concepts that will help you create your own applications.
```
Comment (kprosise) [inline source/tutorials/getting-started-with-docker/getting-started-with-docker.rst:8]:
```suggestion
with the basic commands and concepts that will help you create your own apps.
```

**Comment by doanac** [comment]
Docs for 50ec9602ce5c887e940fc97a57369a0ea4331eb5 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2225/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 2658e512291667fcfc2d0e90dd19633699793f26 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2226/docs/artifacts/html/index.html

**Pull request merged**